doc/entities.md

```diff
@@ -52,20 +52,25 @@ There's an easier way to do this, as described in the chapter [Configuration](co
 Here's an example that showcases (almost) all possibilities for your entity:
 
 ```js
-import {ensure} from 'aurelia-validation';
-import {Entity, type, association, validatedResource} from 'aurelia-orm';
+import {Entity, type, association, resource} from 'aurelia-orm';
+import {ValidationRules} from 'aurelia-validation';
 
-@validatedResource('product')
+@resource('product')
 export class Product extends Entity {
   @type('string')
-  @ensure(it => it.isNotEmpty().containsOnlyAlpha().hasLengthBetween(3, 20))
   name = null;
 
   @association('category')
   category = null;
 
   @association({collection: 'media'})
   media = [];
+
+  enableValidation() {
+    ValidationRules
+      .ensure(a => a.name).required()
```

Comment:
hehe, not sure what 'a' actually is. but in the swan sample i change to the other syntax of
`ensure('name')`will do that here as well